docs/networks.md

```diff
@@ -3,14 +3,13 @@
 
 Networks in the HaaS have three fields:
 
-- 'creator' represents what project created it; or 'admin' if it was
-  administrator-created.  If there are still networks created by a project,
-  then that project cannot be deleted.  Only the creator has the ability to
+- 'owner' represents what project owns it; or 'admin' if it was
```

Comment:
I don't have an objection, but am not sure I understand the distinction here. I didn't think python recognized `null` so if it was inputting/outputting that it would either need to be `None` or the string `'null'`. Would it be handled in the same way that `None` is the access for public networks?